### DISCUSSION TITLE
在vsto开发中使用handyControl

### DISCUSSION THREAD

**Discussion by gukc007** [HandyOrg/HandyControl#602] [Q&A] [answered]
引用控件时比如
        <hc:TabControl>
            <hc:TabItem Header="23"></hc:TabItem>
            <hc:TabItem Header="23"></hc:TabItem>
        </hc:TabControl>

一加载就报错
![image](https://user-images.githubusercontent.com/30338423/103127627-77f95100-46cd-11eb-843c-535eb48e146c.png)

**Comment by ghost1372** [comment, marked as answer]
see here
https://github.com/HandyOrg/HandyControl/issues/603#issuecomment-761543055
